cs-cz/javascript.html.markdown

```diff
@@ -65,30 +60,30 @@ delejNeco()
 18.5 % 7; // = 4.5
 
 // Bitové operace také fungují; když provádíte bitové operace, desetinné číslo  
-// (float) se převede na celé číslo (int) se znaménkem *do* 32 bitů
+// (float) se převede na celé číslo (int) se znaménkem *až do* 32 bitů
 1 << 2; // = 4
 
 // Přednost se vynucuje závorkami.
 (1 + 3) * 2; // = 8
 
-// Existují 3 hodnoty mimo obor reálných čísel
+// Existují 3 hodnoty mimo obor reálných čísel:
 Infinity; // + nekonečno; výsledek např. 1/0
 -Infinity; // - nekonečno; výsledek např. -1/0
 NaN; // výsledek např. 0/0, znamená, že výsledek není číslo ('Not a Number')
 
-// Také existují hodnoty typu bool
+// Také existují hodnoty typu boolean.
 true; // pravda
 false; // nepravda
 
-// Řetězce znaků jsou obaleny  ' nebo ".
+// Řetězce znaků jsou obaleny ' nebo ".
 'abc';
-"Ahoj světe!";
+"Hello, world";
```

Comment:
Why so? It seems, JS supports input in all languages.

Reply:
`Hello world` is not usually being translated to Czech as it is generally know term. It is used in tutorials for beginners in programming. I think that this text targets audience with at least some experience. So I think `Hello world` fits better.